langlib/lang.function/src/main/java/org/ballerinalang/langlib/function/Call.java

```diff
@@ -64,7 +64,9 @@ public static Object call(BFunctionPointer<Object, Object> func, Object... args)
             throw ErrorCreator.createError(
                         getModulePrefixedReason(FUNCTION_LANG_LIB, INCOMPATIBLE_ARGUMENTS),
                         BLangExceptionHelper.getErrorDetails(RuntimeErrors.INCOMPATIBLE_ARGUMENTS,
-                        new BTupleType(paramTypes, restType, 0, false), new BTupleType(argTypes)));
+                        new BTupleType(paramTypes, restType, 0, false)
+                                .toString().replace("[", "").replace("]", ""),
+                        new BTupleType(argTypes).toString().replace("[", "").replace("]", "")));
```

Comment:
Can do a substring instead? Let's also extract the substring logic out to a method.